File: Java/Basics To Java/src/B/B8.java
package B;

import java.util.Scanner;

public class B8 {
	public static void mainx(String[] args) {
		Scanner sc = new Scanner(System.in);

		System.out.println("zadej znak");
		char znak = sc.next().charAt(0);

		if (znak >= 48 && znak <= 57)
			System.out.println("znak je èíslo");
		else if (znak >= 65 && znak <= 90)
			System.out.println("znak je velké písmeno");
		else if (znak >= 97 && znak <= 122)
			System.out.println("znak je malé písmeno");
	}
}

File: C#/Basic logic/B/B8.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace B
{
    internal class B8
    {
        static void Mainx(string[] args)
        {


            Console.WriteLine("zadej jakýkoliv znak");
            char znak = char.Parse(Console.ReadLine());

            if (znak >= 48 && znak<=57)
                Console.WriteLine("znak je číslo");
            else if (znak >=65 && znak<=90)
                Console.WriteLine("znak je velké písmeno");
            else if (znak>=97 && znak<=122)
                Console.WriteLine("znak je malé písmeno");

        }

    }
}
